run_kraken.py

```diff
@@ -67,9 +67,9 @@ def main(cfg):
 
         # Cluster info
         logging.info("Fetching cluster info")
-        cluster_version = runcommand.invoke("kubectl get clusterversion")
+        cluster_version = runcommand.invoke("kubectl get clusterversion", 60)
```

Comment:
I have hit an issue here before where I am running kraken on a different cluster than the one set using export KUBECONFIG... and it gives me the wrong clusterversion and cluster info
We should use --kubeconfig on all these commands to properly execute on the correct kubeconfig we are going to run kraken on

Reply:
Nice catch @paigerube14, will fix it to explicitly use the config defined in the kraken config.

Reply:
We now have the KUBECONFIG env var set to make sure we are using the cluster specified in the kraken config. PTAL. Thanks.